sda-mq/templates/statefulset.yaml

```diff
@@ -143,4 +143,8 @@ spec:
     {{- if .Values.persistence.storageClass }}
       storageClassName: {{ .Values.persistence.storageClass | quote}}
     {{- end }}
+  {{- else if and .Values.persistence.enabled .Values.persistence.nfs }}
+        nfs:
+          server: {{ required "An MQ NFS server is required" .Values.persistence.nfsServer | quote }}
+          path: {{ if .Values.persistence.nfsPath }}{{ .Values.persistence.nfsPath | quote}}{{ else }}{{ "/" }}{{ end }}
```

Comment:
Two nested ifs would make the chart clearer imo.
```
{{- if and .Values.persistence.enabled }}
   {{- if and .Values.persistence.existingClaim (not .Values.persistence.nfs) }}
.
.
  {{- end }}
{{- end }}
```

Reply:
was trying to keep it consistent with the other templates but can do.

Reply:
Then I think its fine as it is